javascript/javascript1/week1/manAge.js

```diff
@@ -0,0 +1,9 @@
+//1. Age-ify (A future age calculator)
+
+let yearOfBirth = 1983;
+let futureYear = 2027;
+
+let age = futureYear - yearOfBirth;
+
+//console.log("You will be " +age+ " years old in " +futureYear);
+console.log(`You will be ${age} years old in ${futureYear}.`);
```

Comment:
Perfect choice - [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are so much cleaner than string concatenations.

Reply:
Thank you..